invisible_cities/cities/hypathia_test.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import tables as tb
+import numpy  as np
+
+from .. core.configure import configure
+from .. core.testing_utils  import assert_tables_equality
+
+from . hypathia import hypathia
+
+
+def test_hypathia_exact_result(ICDATADIR, output_tmpdir):
+    file_in     = os.path.join(ICDATADIR    ,      "Kr83_nexus_v5_03_00_ACTIVE_7bar_3evts.MCRD.h5")
+    file_out    = os.path.join(output_tmpdir,                          "exact_result_hypathia.h5")
+    true_output = os.path.join(ICDATADIR    , "Kr83_nexus_v5_03_00_ACTIVE_7bar_3evts.hypathia.h5")
+
+    conf = configure("hypathia invisible_cities/config/hypathia.conf".split())
+    conf.update(dict(run_number   = -6340,
+                     files_in     = file_in,
+                     file_out     = file_out,
+                     event_range  = (0, 2)))
+
+    # Set a specific seed because we want the result to be
+    # repeatible. Go back to original state after running.
+    original_random_state = np.random.get_state()
+    np.random.seed(123456789)
+    hypathia(**conf)
+    np.random.set_state(original_random_state)
+
+    tables = (     "MC/extents"    ,      "MC/hits"      ,    "MC/particles", "MC/generators",
```

Comment:
The MC table structure is being changed so with PR #693 the comparison of the MC tables in the output is more complex. I'd just remove them from the tables tuple here since the copy_mc_info function is tested separately and this would just have to be changed anyway

Reply:
I'm not sure about the status of #693. If you think the approval will be very soon, I can remove those tables from the check here. If not, maybe it would be better to remove them in a commit in #693 after rebasing on top on this one (assuming this approval won't take long either).

Reply:
OK, just leave it for now, we can change it in the other PR. Hopefully it'll get approved really soon but we shouldn't rely on other PRs for this reason.